web/vue/src/components/gekko/singleStratrunner.vue

```diff
@@ -57,15 +57,15 @@
                 .grd-row-col-3-6 {{ round(report.balance) }}
               .grd-row
                 .grd-row-col-3-6 Market
-                .grd-row-col-3-6 {{ round(report.market) }} {{ data.watch.currency }}
+                .grd-row-col-3-6 {{round(report.market / 100 * report.startPrice)}} {{ data.watch.currency }} ({{ round(report.market) }} %)
```

Comment:
This now describes market as notional with % behind it? Sounds cool!

Reply:
yep absolute value in currency en relative in %